config/locales/en.yml

```diff
@@ -797,6 +797,9 @@ en:
         preamble: Provide in-depth information about how the server is operated, moderated, funded.
         rules_hint: There is a dedicated area for rules that your users are expected to adhere to.
         title: About
+      allow_referrer_origin:
+        desc_html: Control referrer header on external links. It is recommended to disable this for small instances that you may not want to expose to the public.
```

Comment:
Really minor: it doesn't include HTML, so it doesn't have to be `desc_html`, and `desc` is fine. (The `_html` suffix is needed for HTML entries)

Reply:
> that you may not want to expose to the public

It's not really about exposing the instance itself but about exposing users of the instance.

Reply:
ah okay, i fixed it. fwiw i was copying `noindex` which uses `desc_html`, so it seems we have some inconsistency here?